### PR TITLE
SelectQuery objects instead of Query

### PR DESCRIPTION
**Pull request by rochamarcelo**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@markstory

**Review by Arhell — COMMENTED** [review]
@markstory

File: en/orm/retrieving-data-and-resultsets.rst [hunk 1]
```diff
@@ -1048,8 +1048,8 @@ a ResultSet::
 Checking if a Query or ResultSet is Empty
 -----------------------------------------
 
-You can use the ``isEmpty()`` method on a Query or ResultSet object to see if it
-has any rows in it. Calling ``isEmpty()`` on a Query object will evaluate the
+You can use the ``isEmpty()`` method on a SelectQuery or ResultSet object to see if it
```
Comment (ADmad) [inline en/orm/retrieving-data-and-resultsets.rst]: You can't call `isEmpty()` on `SelectQuery` anymore as the proxying of resultset methods from the query class has been removed.

Comment (rochamarcelo) [reply en/orm/retrieving-data-and-resultsets.rst]: Removed the reference to isEmpty, kept the one using ResultSet

**Pull request merged**